components/raftstore-v2/src/operation/ready/apply_trace.rs

```diff
@@ -609,6 +648,15 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
                     lb.put_flushed_index(region_id, CF_RAFT, tablet_index, admin_flush)
                         .unwrap();
                     ctx.engine.consume(&mut lb, true).unwrap();
+                    let epoch = ctx
+                        .engine
+                        .get_region_state(region_id, admin_flush)
+                        .unwrap()
+                        .unwrap()
+                        .get_region()
+                        .get_region_epoch()
+                        .clone();
+                    self.storage_mut().set_flushed_epoch(epoch);
```

Comment:
why is this necessary? the cluster is about to close.

Reply:
yes, stale sst will delete after flush. I wiil remove it.